morpheus/llm/nodes/prompt_template_node.py

```diff
@@ -26,20 +27,24 @@ class PromptTemplateNode(LLMNodeBase):
 
     def __init__(self, template: str, template_format: typing.Literal["f-string", "jinja"] = "f-string") -> None:
```

Comment:
Missing documentation on this node and all public members. If sphinx can pull documentation from inherited base classes, we can omit documenting `get_input_names()` and `execute()`